src/main/java/switchblade/Storage.java

```diff
@@ -0,0 +1,110 @@
+package switchblade;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * switchblade.Storage will be the class used to handle any IO from switchblade.SwitchBlade to its storage file
+ *
+ * @author leeyueyang
+ */
+
+public class Storage {
+
+    /**
+     * Retrieves and interprets information from "switchblade.SwitchBlade.txt"
+     *
+     * @return Tasks read from switchblade.SwitchBlade.txt file
+     * @param directory String of directory containing "switchblade.SwitchBlade.txt"
+     * @exception FileNotFoundException "switchblade.SwitchBlade.txt" which contains the data has not been created
+     */
+    public static ArrayList<Task> retrieve(String directory) throws FileNotFoundException {
+        String filepath = directory + "/switchblade.SwitchBlade.txt";
+        File file = new File(filepath);
+        ArrayList<Task> output = new ArrayList<>();
```

Comment:
Should plural form be used to represent a collection of objects for ArrayList<Task>?

Reply:
Just want to say thanks for reviewing my code! I've managed to edit my code to make the necessary amends!